beacon_node/operation_pool/src/attestation_storage.rs

```diff
@@ -158,8 +315,9 @@ impl<E: EthSpec> AttestationMap<E> {
         // NOTE: this is sub-optimal and in future we will remove this in favour of max-clique
         // aggregation.
         let mut aggregated = false;
+
         for existing_attestation in attestations.iter_mut() {
-            if existing_attestation.signers_disjoint_from(&indexed) {
+            if existing_attestation.should_aggregate(&indexed) {
                 existing_attestation.aggregate(&indexed);
```

Comment:
`aggregate` here returns an `Option` denoting whether it was actually aggregated. The option isn't used though, so I updated this here: ab819dcdb07ba57c308b5198eac5ad95e303e824